src/menu-button/MenuButton.module.scss

```diff
@@ -0,0 +1,5 @@
+:local {
+  .button {
```

Comment:
Another option would be to use `<Spacer marginRight="small-1">Whatever needs some spacing</Spacer>`
This will add 5px margin instead of 3 though.